Framework/API/inc/MantidAPI/WorkspaceFactory.h

```diff
@@ -77,8 +78,8 @@ class MANTID_API_DLL WorkspaceFactoryImpl
                               const size_t &NVectors, const size_t &XLength,
                               const size_t &YLength) const;
 
-  void initializeFromParent(const MatrixWorkspace_const_sptr parent,
-                            const MatrixWorkspace_sptr child,
+  void initializeFromParent(const MatrixWorkspace &parent,
```

Comment:
Good catch changing `parent` to be a `const ref` can we do the same with `child`?